package.json

```diff
@@ -9,7 +9,10 @@
     "babel:cjs": "nimbus babel --clean --workspaces=\"@superset-ui/!(demo|generator-superset)\"",
     "babel:esm": "nimbus babel --clean --workspaces=\"@superset-ui/!(demo|generator-superset)\" --esm",
     "build:assets": "node ./scripts/buildAssets.js",
-    "clean": "rm -rf ./packages/**/{lib,esm}",
+    "demo": "cd packages/demo && yarn demo:build",
+    "storybook": "cd packages/demo && yarn storybook",
+    "sb": "yarn storybook",
+    "clean": "rm -rf ./{packages,plugins}/**/{lib,esm,tsbuild.info}",
```

Comment:
Clean up `tsbuild.info` too so `tsc` would not be too confused when we move files around.